src/mobile/index.js

```diff
@@ -1,4 +1,13 @@
 import { AppRegistry } from 'react-native';
 import Wallet from './src/ui/routes/entry';
+import { isAndroid } from './src/libs/device';
+
+if (isAndroid) {
+    // This Intl API is not available in android 
+    // It needs to be polyfilled
+    require('intl');
+
+    require('intl/locale-data/jsonp/en-US');
```

Comment:
@rajivshah3 I guess we might need to require all supported locales for Trinity here?

Reply:
Is the `require('intl')` sufficient? The readme says this:

> In Node.js, or when using require('intl'), the data is automatically added to the runtime and does not need to be provided.

https://www.npmjs.com/package/intl#locale-data

Reply:
No it is not. Doesn't work if I remove this import.

Reply:
Oh ok 🤔 yeah I guess we'll need to add all supported locales. Are we planning to release an update before #2012 is merged?

Reply:
I don't think so. Perhaps we should merge this then to fix the crash?